apps/client/src/intros/shared/DynamicIMTRIntro.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Question: do you want to merge #475 before this PR or after?

Reply:
I'll merge that one into this one and then merge this into develop. Good question 👍